### ISSUE TITLE
Make 'Pay for removal' a togglable option

### ISSUE THREAD

**Issue by othyn** [Speaax/Farming-Helper#15]
I have most of this completed already. The bit that isn't is the required items check for the axes, as at the moment there isn't a system in place for only requiring one item from a given set of items.

- [ ] Add a new user preference
- [ ] Wrap the coins tally requirement behind the config check in getAllItemRequirements
- [ ] Wrap the teleport overlay around the config check, to highlight either the farmers or patches
- [ ] Add an enum containing all axes
- [ ] Add an axe requirement to the getAllItemRequirements check
- [ ] Fix the axe requirements, as currently it requires ALL axes as an AND operation as part of the current item requirements system implementation. There needs to be support for ANY axes via a new OR operation for item requirements, in item sets that can be defined as OR sets.

**Comment by Speaax** [comment]
Not gonna lie, the item requirement code scares me :d

**Comment by othyn** [comment]
Yeah I'm not even sure if this is possible lmao, at least not in its current form. The level of permutations that require checking really requires some level of abstraction on the item requirement code first.
